obspy/clients/fdsn/mass_downloader/download_helpers.py

```diff
@@ -349,19 +349,29 @@ def sanitize_downloads(self, logger):
         of the download helpers does not allow for a StationXML file with no
         data.
         """
+        from obspy.io.mseed.util import get_start_and_end_time
         # All or nothing for each channel.
         for id in self.miss_station_information.keys():
-            logger.warning("No station information could be downloaded for "
-                           "%s.%s.%s.%s. All downloaded MiniSEED files "
+            logger.warning("Station information could not be downloaded for "
+                           "%s.%s.%s.%s. MiniSEED files outside of the "
+                           "station information period "
                            "will be deleted!" % (
```

Comment:
On a side note, it is not necessary to `%`-format the string here, because `logger.warning` will take its positional arguments (after the first) and `%`-format them into the message string.